libgnucash/engine/gnc-commodity.cpp

```diff
@@ -188,54 +188,75 @@ static QuoteSourceList currency_quote_sources =
  */
 static QuoteSourceList single_quote_sources =
 {
-    { false, SOURCE_SINGLE, "Alphavantage, US", "alphavantage" },
-    { false, SOURCE_SINGLE, "Amsterdam Euronext eXchange, NL", "aex" },
-    { false, SOURCE_SINGLE, "Association of Mutual Funds in India", "amfiindia" },
-    { false, SOURCE_SINGLE, "Australian Stock Exchange, AU", "asx" },
-    { false, SOURCE_SINGLE, "Canada Mutual", "canadamutual" },
-    { false, SOURCE_SINGLE, "Deka Investments, DE", "deka" },
-    { false, SOURCE_SINGLE, "Dutch", "dutch" },
-    { false, SOURCE_SINGLE, "DWS, DE", "dwsfunds" },
-    { false, SOURCE_SINGLE, "Financial Times Funds service, GB", "ftfunds" },
-    { false, SOURCE_SINGLE, "Finanzpartner, DE", "finanzpartner" },
-    { false, SOURCE_SINGLE, "GoldMoney spot rates, JE", "goldmoney" },
-    { false, SOURCE_SINGLE, "Google Web, US Stocks", "googleweb" },
-    { false, SOURCE_SINGLE, "India Mutual", "indiamutual" },
-    { false, SOURCE_SINGLE, "Morningstar, GB", "morningstaruk" },
-    { false, SOURCE_SINGLE, "Morningstar, JP", "morningstarjp" },
-    { false, SOURCE_SINGLE, "New Zealand stock eXchange, NZ", "nzx" },
-    { false, SOURCE_SINGLE, "Paris Stock Exchange/Boursorama, FR", "bourso" },
-    { false, SOURCE_SINGLE, "Romania", "romania" },
-    { false, SOURCE_SINGLE, "SIX Swiss Exchange shares, CH", "six" },
-    { false, SOURCE_SINGLE, "Skandinaviska Enskilda Banken, SE", "seb_funds" },
-    { false, SOURCE_SINGLE, "Sharenet, ZA", "za" },
-    { false, SOURCE_SINGLE, "TIAA-CREF, US", "tiaacref" },
-    { false, SOURCE_SINGLE, "Toronto Stock eXchange, CA", "tsx" },
-    { false, SOURCE_SINGLE, "T. Rowe Price", "troweprice" },
-    { false, SOURCE_SINGLE, "T. Rowe Price, US", "troweprice_direct" },
-    { false, SOURCE_SINGLE, "Union Investment, DE", "unionfunds" },
-    { false, SOURCE_SINGLE, "US Govt. Thrift Savings Plan", "tsp" },
-    { false, SOURCE_SINGLE, "Yahoo as JSON", "yahoo_json" },
-    { false, SOURCE_SINGLE, "Yahoo Web", "yahooweb" },
-    { false, SOURCE_SINGLE, "YH Finance (FinanceAPI)", "financeapi" },
+    { false, SOURCE_SINGLE, C_("FQ Source", "Alphavantage"), "alphavantage" },
```

Comment:
No, you want `NC_("FQ Source", "Alphavantage"), "alphavantage"`
and then you have to search each usage and decide translate it —`C_("FQ Source", "…")`I think — (shown to the user) or not (stored in the file or log files)

Reply:
I used NC for all as suggested. However, my time has become more limited now and not sure if I will have enough of it to do anything more involved. Given that if this is not what you've been looking for, I propose to remove the translation markers, merge just the updated list and proceed with translations as a separate PR.

Reply:
It's not so hard: The user name is retrieved in exactly one place, so to perform the translation you need only do
```
--- a/gnucash/gnome-utils/dialog-commodity.cpp
+++ b/gnucash/gnome-utils/dialog-commodity.cpp
@@ -778,7 +778,7 @@ gnc_ui_source_menu_create(QuoteSourceType type)
             supported = gnc_quote_source_get_supported(source);
             gtk_list_store_append(store, &iter);
             gtk_list_store_set(store, &iter,
-                               SOURCE_COL_NAME, name,
+                               SOURCE_COL_NAME, C_("FQ Source", name),
                                SOURCE_COL_FQ_SUPPORTED, supported,
                                -1);
```

Reply:
@jralls Thanks for explaining this. I think I begin to understand  how the i18n works in GTK. I've taken inspiration from 11458bf8b37f874fa5929d269814e9cb6da570f6 and used `g_dpgettext2` to avoid string concatenation.